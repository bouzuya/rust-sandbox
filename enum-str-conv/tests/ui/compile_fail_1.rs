fn main() {
    #[derive(enum_str_conv::EnumStrConv)]
    struct MyStruct;
}
