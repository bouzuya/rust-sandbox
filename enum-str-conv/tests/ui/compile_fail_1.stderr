error: EnumStrConv can only be derived for enums
 --> tests/ui/compile_fail_1.rs:3:5
  |
3 |     struct MyStruct;
  |     ^^^^^^^^^^^^^^^^
